apps/examples/smartfs/smart/smart_main.c

```diff
@@ -16,7 +16,7 @@
  *
  ****************************************************************************/
 /****************************************************************************
- * examples/smart/smart_main.c
+ * apps/examples/smartfs/smart/smart_main.c
```

Comment:
What's difference to fs_test?

Reply:
Sorry, I missed this comment. I will update this.
Thank you!